zp-relayer/validateTx.ts

```diff
@@ -27,7 +28,7 @@ type OptionError = Error | null
 export async function checkAssertion(f: () => Promise<OptionError> | OptionError) {
   const err = await f()
   if (err) {
-    logger.error('Assertion error: %s', err.message)
+    logger.warn('Assertion error: %s', err.message)
     throw err
```

Comment:
Will `throw` cause the stack trace in the logs?

Reply:
As all validation checks are performed in workers only, this `throw` will be caught by worker runner.
Log example with incorrect transfer proof:
```
debug: Added job: 2 {"timestamp":"2022-09-29T21:14:20.175Z"}
info: POOL WORKER: Job 2: processing... {"timestamp":"2022-09-29T21:14:20.180Z"}
info: Recieved 1 txs {"timestamp":"2022-09-29T21:14:20.180Z"}
debug: Fee: 0 {"timestamp":"2022-09-29T21:14:20.189Z"}
warn: Assertion error: Incorrect transfer proof {"timestamp":"2022-09-29T21:14:20.205Z"}
info: Updated gasPrice: { gasPrice: '30000000013' } {"timestamp":"2022-09-29T21:14:22.104Z"}
```

Reply:
thanks